src/reactive/extMethods/streamFor.js

```diff
@@ -30,17 +35,17 @@ Observable.prototype.observeOn = function(router, modelId) {
                 if(hasCompleted ) {
                     return;
                 }
-                router.runAction(modelId, () => {
+                source._router.runAction(modelId, () => {
```

Comment:
it's a 'private' var on `RouterObservable` so accessing it via a prototype method for that type is reasonable